packages/react-magma-dom/src/components/Table/TablePagination.tsx

```diff
@@ -108,6 +114,10 @@ const StyledContainer = styled.div<{
   display: flex;
   justify-content: flex-end;
   padding: ${props => props.theme.spaceScale.spacing02};
+  border-radius: ${props => 
+    props.hasSquareCorners 
+    ? "0;"
+    : `0 0 ${props.theme.borderRadius} ${props.theme.borderRadius};`}
```

Comment:
nitpick but for consistency:

```suggestion
  border-radius: ${props => 
    props.hasSquareCorners 
    ? "0"
    : `0 0 ${props.theme.borderRadius} ${props.theme.borderRadius}`};
```